types/scte224v20200407/sctestructures.go

```diff
@@ -0,0 +1,325 @@
+package scte224v20200407
+
+import (
+	"encoding/xml"
+	"log"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Comcast/scte224structs/types/scte224v20180501/adi30"
+)
+
+const schemaLocation = "https://www.scte.org/standards-development/library/standards-catalog/ansiscte-224-2018r1/"
```

Comment:
how durable is this link likely to be?

Reply:
Not quite sure, but I did got to this link through the search box on that website. Valid point though, I see that, 2018 link is now not working